SimCalorimetry/HcalTrigPrimAlgos/interface/HcalFinegrainBit.h

```diff
@@ -27,13 +27,13 @@ class HcalFinegrainBit {
 
   // define prompt-delayed TDC range. Note this is offset from depth and ieta by 1
   const int tdc_boundary[29][7] = {
-      {8, 14, 15, 17, 0, 0, 0}, {8, 14, 15, 17, 0, 0, 0}, {8, 14, 14, 17, 0, 0, 0}, {8, 14, 14, 17, 0, 0, 0},
-      {8, 13, 14, 16, 0, 0, 0}, {8, 13, 14, 16, 0, 0, 0}, {8, 12, 14, 15, 0, 0, 0}, {8, 12, 14, 15, 0, 0, 0},
-      {7, 12, 13, 15, 0, 0, 0}, {7, 12, 13, 15, 0, 0, 0}, {7, 12, 13, 15, 0, 0, 0}, {7, 12, 13, 15, 0, 0, 0},
-      {7, 11, 12, 14, 0, 0, 0}, {7, 11, 12, 14, 0, 0, 0}, {7, 11, 12, 14, 0, 0, 0}, {7, 11, 12, 7, 0, 0, 0},
-      {0, 12, 10, 0, 0, 0, 0},  {0, 9, 10, 9, 10, 0, 0},  {16, 9, 9, 9, 11, 10, 0}, {17, 8, 9, 8, 9, 10, 0},
-      {9, 7, 7, 7, 9, 6, 0},    {8, 7, 7, 6, 6, 6, 0},    {8, 6, 6, 6, 7, 7, 0},    {7, 6, 6, 6, 7, 6, 0},
-      {7, 6, 6, 6, 6, 6, 0},    {6, 6, 6, 6, 6, 6, 0},    {6, 5, 6, 6, 6, 7, 10},   {9, 9, 9, 5, 5, 6, 6},
+      {12, 12, 12, 12, 0, 0, 0}, {12, 12, 12, 12, 0, 0, 0}, {12, 12, 12, 12, 0, 0, 0}, {12, 12, 12, 12, 0, 0, 0},
+      {12, 12, 12, 12, 0, 0, 0}, {12, 12, 12, 12, 0, 0, 0}, {12, 12, 12, 12, 0, 0, 0}, {12, 12, 12, 12, 0, 0, 0},
+      {12, 12, 12, 12, 0, 0, 0}, {12, 12, 12, 12, 0, 0, 0}, {12, 12, 12, 12, 0, 0, 0}, {12, 12, 12, 12, 0, 0, 0},
+      {12, 12, 12, 12, 0, 0, 0}, {12, 12, 12, 12, 0, 0, 0}, {12, 12, 12, 12, 0, 0, 0}, {12, 12, 12, 12, 0, 0, 0},
+      {0, 12, 10, 0, 0, 0, 0},   {0, 9, 10, 9, 10, 0, 0},   {16, 9, 9, 9, 11, 10, 0},  {17, 8, 9, 8, 9, 10, 0},
+      {9, 7, 7, 7, 9, 6, 0},     {8, 7, 7, 6, 6, 6, 0},     {8, 6, 6, 6, 7, 7, 0},     {7, 6, 6, 6, 7, 6, 0},
+      {7, 6, 6, 6, 6, 6, 0},     {6, 6, 6, 6, 6, 6, 0},     {6, 5, 6, 6, 6, 7, 10},    {9, 9, 9, 5, 5, 6, 6},
```

Comment:
My expectation is that this won't be changed frequently. I anticipate these LUTs (hopefully) being constant for much of Run 3. So for now, this is to have the MC / emulation agree with the deployed HCAL settings, and shouldn't need revisions. I agree that in the future, it would be more ideal to have an implementation of these TDC LUTs that is configured or referenced based on the on-detector settings.

Reply:
Alright, good enough for me for now. It would be nice to have a more definite configuration plan for the future.